homeassistant/components/media_player/emby.py

```diff
@@ -234,18 +251,31 @@ def media_duration(self):
                 return None
 
     @property
+    def media_position(self):
+        """Position of current playing media in seconds."""
+        return self.media_status_last
+
+    @property
+    def media_position_updated_at(self):
+        """
+        When was the position of the current playing media valid.
+
+        Returns value from homeassistant.util.dt.utcnow().
+        """
+        return self.media_status_received
+
+    @property
     def media_image_url(self):
         """Image url of current playing media."""
         if self.now_playing_item is not None:
             try:
                 return self.client.get_image(
-                    self.now_playing_item['ThumbItemId'], 'Thumb',
-                    self.play_percent())
+                    self.now_playing_item['ThumbItemId'], 'Thumb', 0)
             except KeyError:
                 try:
                     return self.client.get_image(
```

Comment:
The get_image function in the emby library only returns a URL based on the emby api syntax.  It doesn't actually make any contact with the emby server.